src/server/scripts/Commands/cs_server.cpp

```diff
@@ -132,7 +132,10 @@ class server_commandscript : public CommandScript
     // Display the 'Message of the day' for the realm
     static bool HandleServerMotdCommand(ChatHandler* handler, char const* /*args*/)
     {
-        handler->PSendSysMessage(LANG_MOTD_CURRENT, Motd::GetMotd());
+        if (handler->IsHumanReadable())
```

Comment:
yea, my example only has that _because_ there's only one arg

named arguments sound like a great idea, but would bloat the PR workload. can we implement that in a follow-up?